### PR TITLE
two-bucket: remove broken link

### PR DESCRIPTION
**Pull request by Smarticles101**

### CODE REVIEW

**Comment by petertseng** [comment]
Simply saying "Lindsay" may be a little ambiguous. Would it be better to say "Lindsay Levine" like https://github.com/exercism/problem-specifications/pull/1093 does?

**Comment by Smarticles101** [comment]
@petertseng great idea

File: exercises/two-bucket/description.md [hunk 1]
```diff
@@ -25,4 +25,4 @@ To conclude, the only valid moves are:
 - emptying one bucket and doing nothing to the other
 - filling one bucket and doing nothing to the other
 
-Written with <3 at [Fullstack Academy](http://www.fullstackacademy.com/) by [Lindsay](http://lindsaylevine.com).
+Written with <3 at [Fullstack Academy](http://www.fullstackacademy.com/) by Lindsay Levine.
```
Comment (rpottsoh) [inline exercises/two-bucket/description.md]: I'm getting a 404 from the fullstackacademy.com url.  An alternative address should be sought out or at least this one removed.

Comment (Smarticles101) [reply exercises/two-bucket/description.md]: The fullstackacademy url works for me

Comment (rpottsoh) [reply exercises/two-bucket/description.md]: I tried it again and it worked this time.  First few attempts were 404.

Comment (Smarticles101) [reply exercises/two-bucket/description.md]: odd

**Comment by petertseng** [comment]
That is a shame. Those commits should have been squashed.

**Comment by Smarticles101** [comment]
oops rip

**Comment by petertseng** [comment]
> this PR does the exact same thing as #1093

and why is this the case? I don't know, I don't really consider it my priority to find out why

**Comment by Smarticles101** [comment]
I checked the commit history and that pull request commit and the merge commit were nowhere to be found

**Comment by petertseng** [comment]
I wonder why! (I do not know why)

**Pull request merged**